extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerConfig.java

```diff
@@ -118,4 +118,9 @@
     @ConfigItem(defaultValue = "true")
     public Boolean logTraceContext;
 
+    /**
+     * Whether jaeger should run in zipkin compatibility mode
+     */
+    @ConfigItem
```

Comment:
I may not have been clear earlier, but this needs to be done as"

```
@ConfigItem(defaultValue = "false")
public Boolean zipkinCompatibilityMode
```

Reply:
I didn't understand that, i changed as you requested.